src/refill.rs

```diff
@@ -223,6 +223,14 @@ mod tests {
         assert_eq!(options.line_ending, LineEnding::LF);
     }
 
+    #[test]
+    fn test_unfill_missing() {
```

Comment:
The word `missing` is confusing here. I think the test is covering the case where a bullet point list is _not_  actually seen as a list?

Reply:
I think you are right and I will manually change the name, thanks.

Reply:
Thanks!